.github/workflows/release_portable_linux_packages.yml

```diff
@@ -208,12 +212,25 @@ jobs:
           python ./build_tools/third_party/s3_management/manage.py ${{ env.S3_SUBDIR }}/${{ matrix.target_bundle.amdgpu_family }}
 
       - name: Trigger testing nightly tarball
-        if: ${{ inputs.build_type == 'rc' || inputs.build_type == '' }}
+        if: ${{ env.RELEASE_TYPE == 'nightly' }}
         uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc # v1.2.4
         with:
           workflow: test_release_packages.yml
           inputs: '{ "version": "${{ needs.setup_metadata.outputs.version }}", "tag": "nightly-tarball", "file_name": "${{ env.FILE_NAME }}", "target": "${{ matrix.target_bundle.amdgpu_family }}" }'
 
+      - name: Trigger building PyTorch wheels
+        if: ${{ github.repository_owner == 'ROCm' }}
+        uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc # v1.2.4
+        with:
+          workflow: release_linux_pytorch_wheels.yml
```

Comment:
So will this trigger the `release_linux_pytorch_wheels` workflow multiple times, once per `matrix` instance? 

> We may want to further revise this later, as there is no clear and easy to follow link between the triggering and the triggered workflow.

Agreed, but I guess we should be pragmatic first.

Reply:
> So will this trigger the `release_linux_pytorch_wheels` workflow multiple times, once per `matrix` instance?

Exactly. I triggered another run with a full matrix:
https://github.com/ROCm/TheRock/actions/runs/16164094877

This triggered `Release Linux PyTorch Wheels` for each architecture:
* https://github.com/ROCm/TheRock/actions/runs/16166448085
* https://github.com/ROCm/TheRock/actions/runs/16167600825
* https://github.com/ROCm/TheRock/actions/runs/16168515469
* https://github.com/ROCm/TheRock/actions/runs/16168764186

gfx942 failed in the above run (docker died).